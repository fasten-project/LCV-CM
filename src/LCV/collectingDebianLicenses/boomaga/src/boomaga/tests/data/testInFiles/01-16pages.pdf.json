{
    "checksum": "cb8e1e9ff482db2717986e60a9e40401c35bdedc6e32bf196509d516eeec1a15",
    "file": "01-16pages.pdf",
    "mime": {
        "encoding": "binary",
        "type": "application/pdf"
    },
    "number_of_duplicates": 1,
    "package": "boomaga",
    "path": "boomaga/3.0.0-1/src/boomaga/tests/data/testInFiles/01-16pages.pdf",
    "pkg_infos": {
        "area": "main",
        "copyright": true,
        "ctags_count": 0,
        "license": "/copyright/license/boomaga/3.0.0-1/",
        "metric": {
            "size": 5604
        },
        "pts_link": "https://tracker.debian.org/pkg/boomaga",
        "sloc": [
            [
                "cpp",
                13687
            ],
            [
                "sh",
                314
            ],
            [
                "xml",
                9
            ],
            [
                "makefile",
                8
            ]
        ],
        "suites": [
            "bookworm",
            "bullseye",
            "sid"
        ],
        "vcs_browser": "https://gitlab.com/perezmeyer/debian-package-boomaga",
        "vcs_type": "git"
    },
    "raw_url": "/data/main/b/boomaga/3.0.0-1/src/boomaga/tests/data/testInFiles/01-16pages.pdf",
    "stat": {
        "perms": "rw-r--r--",
        "size": 8530,
        "symlink_dest": null,
        "type": "-"
    },
    "text_file": false,
    "type": "file",
    "version": "3.0.0-1"
}