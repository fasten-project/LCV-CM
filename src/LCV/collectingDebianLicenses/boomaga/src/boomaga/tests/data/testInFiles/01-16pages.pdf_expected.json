{
    "checksum": "359db082e350730096ff2f9966cc7682f6de3085816694d01980a95d0ad5681b",
    "file": "01-16pages.pdf_expected",
    "mime": {
        "encoding": "us-ascii",
        "type": "text/plain"
    },
    "number_of_duplicates": 2,
    "package": "boomaga",
    "path": "boomaga/3.0.0-1/src/boomaga/tests/data/testInFiles/01-16pages.pdf_expected",
    "pkg_infos": {
        "area": "main",
        "copyright": true,
        "ctags_count": 0,
        "license": "/copyright/license/boomaga/3.0.0-1/",
        "metric": {
            "size": 5604
        },
        "pts_link": "https://tracker.debian.org/pkg/boomaga",
        "sloc": [
            [
                "cpp",
                13687
            ],
            [
                "sh",
                314
            ],
            [
                "xml",
                9
            ],
            [
                "makefile",
                8
            ]
        ],
        "suites": [
            "bookworm",
            "bullseye",
            "sid"
        ],
        "vcs_browser": "https://gitlab.com/perezmeyer/debian-package-boomaga",
        "vcs_type": "git"
    },
    "raw_url": "/data/main/b/boomaga/3.0.0-1/src/boomaga/tests/data/testInFiles/01-16pages.pdf_expected",
    "stat": {
        "perms": "rw-r--r--",
        "size": 49,
        "symlink_dest": null,
        "type": "-"
    },
    "text_file": true,
    "type": "file",
    "version": "3.0.0-1"
}